src/components/button/daikin-button.ts

```diff
@@ -32,8 +32,13 @@ const cvaButton = cva(
     "rounded",
     "tracking-wide",
     "text-nowrap",
+
+    "enabled:focus-visible:outline",
+    "enabled:focus-visible:outline-1",
+    "enabled:focus-visible:outline-offset-1",
+    "enabled:focus-visible:outline-daikinBlue-700",
```

Comment:
I don't think we need `enabled:` as disabled buttons cannot have focus.

```suggestion
    "focus-visible:outline",
    "focus-visible:outline-1",
    "focus-visible:outline-offset-1",
    "focus-visible:outline-daikinBlue-700",
```